kedro_neptune/__init__.py

```diff
@@ -100,37 +101,41 @@ def init(metadata: ProjectMetadata, api_token: str, project: str, base_namespace
             }, config_file, default_flow_style=False)
 
 
-class NeptuneMetadataDataSet(AbstractDataSet):
-    def __init__(self, run_id: str):
-        self._run_id: str = run_id
-        self._base_namespace: str = 'kedro'
-        self._run: Optional[neptune.Run] = None
+def get_neptune_config():
+    session: KedroSession = get_current_session()
+    context = session.load_context()
+    # pylint: disable=protected-access
+    credentials = context._get_config_credentials()
+    config = context.config_loader.get('neptune**')
 
-    def _save(self, data: Dict[str, Any]) -> None:
-        assert self._run is not None
+    api_token = credentials['neptune']['NEPTUNE_API_TOKEN']
+    project = config['neptune']['project']
+    base_namespace = config['neptune']['base_namespace'] or 'kedro'
+    source_files = config['neptune']['upload_source_files']
+
+    return api_token, project, base_namespace, source_files
 
-        self._run.assign(data)
+
+class NeptuneMetadataDataSet(AbstractDataSet):
+    def _save(self, data: Dict[str, Any]) -> None:
+        pass
 
     def _describe(self) -> Dict[str, Any]:
-        return getattr(self._run, 'get_structure', {})
+        pass
```

Comment:
I don't like these empty implementations -- looks a great opportunity to fail silently, which is terrible to debug